tests/apps/hostel/factories.py

```diff
@@ -0,0 +1,71 @@
+from apps.common.schemas import BaseInSchema
```

Comment:
local imports should be below imports of 3-rd party libs

Reply:
ok